src/RadixSortLSD.chpl

```diff
@@ -25,6 +13,20 @@ module RadixSortLSD
     private config const logLevel = ServerConfig.logLevel;
     const rsLogger = new Logger(logLevel);
 
+    record RadixSortLSDPlan {
```

Comment:
I would also add/prefer a factory function called `makeRadixSortLSDPlan()` which creates and returns a plan, that way in the future we can do more than just `init()` then plan.

Reply:
it took me a min to figure out how this was different that using the constructor, but i think i understand now.

So for the time being, it'll be equivalent to initializing a plan, but in the future it could return a preexisting plan with the same configuration (which was cached from a previous sort)

am i understanding this correctly?

Reply:
yes, and it could throw exceptions which you can't do in a `class`es `init()`